compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -32,6 +33,30 @@
 from compiler_opt.rl import env
 
 
+@dataclasses.dataclass
+class SequenceExampleFeatureNames:
+  """Feature names for features that are always added to seq example."""
+  action: str = 'action'
+  reward: str = 'reward'
+  module_name: str = 'module_name'
+
+
+def get_loss(seq_example: tf.train.SequenceExample,
```

Comment:
neat, I wonder if we should move the sequence example stuff in its own utility library? if you think that's a good idea, can you tag an issue on this, we can do it after.